infer_segmentation.py

```diff
@@ -344,5 +361,20 @@ def main(gcp_bucket, model_id, background_class_index, stack_id, image_ids, user
         type=str,
         default=None,
         help='The specified trained thresholds file id.')
+    argparser.add_argument(
+        '--python-random-global-seed',
+        type=str,
+        default='1',
+        help='The  setting of random.seed(global seed), where global seed is int convertible or None.')
+    argparser.add_argument(
+        '--numpy-random-global-seed',
+        type=str,
+        default='12',
+        help='The setting of np.random.seed(global seed), where global seed is int convertible or None.')
+    argparser.add_argument(
+        '--tf-random-global-seed',
+        type=str,
+        default='123',
+        help='The setting of (from tensorflow) set_random_seed(global seed), where global seed is int convertible or None.')
```

Comment:
hmm this may be easier to us an `int` type rather than a `str` (also clearer since the value is actually an int :))- or were you hitting some other issue?

Reply:
see above